src/main/java/keycontacts/model/Model.java

```diff
@@ -78,11 +82,17 @@ public interface Model {
     void setStudent(Student target, Student editedStudent);
 
     /** Returns an unmodifiable view of the filtered student list */
-    ObservableList<Student> getFilteredStudentList();
+    ObservableList<Student> getStudentList();
 
     /**
      * Updates the filter of the filtered student list to filter by the given {@code predicate}.
      * @throws NullPointerException if {@code predicate} is null.
      */
-    void updateFilteredStudentList(Predicate<Student> predicate);
+    void updateStudentList(Predicate<Student> predicate);
+
+    /**
+     * Updates the sorted student list to sort by the given {@code comparator}.
+     * @throws NullPointerException if {@code comparator} is null.
+     */
+    void updateStudentList(Comparator<Student> comparator);
```

Comment:
Would it be clearer to change these to `filterStudentList` and `sortStudentList`

Reply:
I tried to maintain what the original codebase names. The method signature is different, so I thought this could work as well.